src/prefect/client/subscriptions.py

```diff
@@ -33,7 +33,15 @@ async def __anext__(self) -> S:
         while True:
             try:
                 await self._ensure_connected()
-                return self.model.parse_raw(await self._websocket.recv())
+                message = await self._websocket.recv()
+
+                message_data = orjson.loads(message)
+
+                if message_data.get("type") == "ping":
+                    await self._websocket.send(orjson.dumps({"type": "pong"}).decode())
+                    continue
```

Comment:
Oh bummer, I thought we could use the built-in `websocket.ping()` but that doesn't seem to be available in the server-side sockets.  This works great too!